src/components/GraphPage/index.js

```diff
@@ -27,21 +27,23 @@ class GraphPage extends Component {
       <section id="graph-page">
         <h3>wonderful women</h3>
         <div className="content">
-          <p>{ `There's a woman behind every single change in our history — whether you know her
-          or not. We have picked a few (for now) to try and show it, by sorting them into their
-          influence areas and through time.` }</p>
+          <p>{ `There's a woman behind every single change in our history — whether we know her or not. We have picked a few to try and show it, by sorting them by their influence fields and through time. Take a look and get surprised by how much girl power history can reveal!` }</p>
         </div>
-        <div className="graph-head">
-          <h4 className={ this.styleTitle(this.props.titlePosition) }>
-            Who are the wonderful women who helped shape the world?
-          </h4>
-          <div className="legend">
-            <span>Women</span>
+        <div className="wrapper">
+          <div className="graph-content">
+          <div className="graph-head">
+              <h3 className={ this.styleTitle(this.props.titlePosition) }>
+                Who are the wonderful women who helped shape the world?
+              </h3>
+              <div className="legend">
+                <span>Women</span>
+              </div>
+            </div>
+            <DataLoader fetchData={this.fetchData} />
+            <FilterableScatterPlot dataList={this.state.dataList} options={this.state.options} />
+            <p className="information">Information: We collected the information presented from public sources like <a href="https://www.wikipedia.org/" target="_blank">Wikipedia</a> and do not take responsability for the content in the external links.</p>
```

Comment:
Also, use the template standard we've been using for long lines of text. (Look at line 30 of `src/components/GraphPage/index.js`)